src/ring_theory/jacobson.lean

```diff
@@ -63,22 +76,13 @@ begin
     exact λ J hJ, hx ⟨le_trans hP.left hJ.left, hJ.right⟩ }
 end
 
-/-- A ring `R` is Jacobson if and only if every radical ideal is the infimum of a collection of maximal ideals. -/
+/-- A ring `R` is Jacobson if and only if for every prime ideal `I`,
+ `I` can be written as the infimum of some collection of maximal ideals.
+ Allowing ⊤ in the set `M` of maximal ideals is equivalent, but makes some proofs cleaner. -/
 lemma is_jacobson_iff_Inf_maximal : is_jacobson R ↔
-    ∀ {I : ideal R}, I.radical = I → ∃ M ⊆ {J : ideal R | J.is_maximal ∨ J = ⊤}, I = Inf M :=
-begin
-  use λ H I h, ⟨{J : ideal R | I ≤ J ∧ J.is_maximal}, ⟨λ _ hJ, or.inl hJ.right, eq.symm (H _ h)⟩⟩,
-  intros H I hI,
-  rcases H hI with ⟨M, hM, hInf⟩,
-  refine le_antisymm _ le_jacobson,
-  intros x hx,
-  rw hInf,
-  erw mem_Inf at ⊢ hx,
-  intros I hI,
-  cases hM hI with is_max is_top,
-  { refine hx ⟨le_Inf_iff.1 (le_of_eq hInf) I hI, is_max⟩ },
-  { rw is_top, exact submodule.mem_top }
-end
+  ∀ {I : ideal R}, I.is_prime → ∃ M : set (ideal R), (∀ J ∈ M, is_maximal J ∨ J = ⊤) ∧ I = Inf M :=
```

Comment:
Since `is_maximal I` is defined as `I ≠ ⊤ ∧ ∀ J, I < J → J = ⊤`, it might be useful to add `is_jacobson_iff_Inf_maximal' : is_jacobson R ↔ ∀ {I : ideal R}, I.is_prime → ∃ M : set (ideal R), (∀ (J ∈ M) (K : ideal R), J < K -> K = ⊤) ∧ I = Inf M`.